tests/views_integration/Tribe/Events/Views/V2/Partials/Month_View/Calendar_Body/Day/Calendar_Events/Calendar_Event/__snapshots__/TooltipTest__test_render_with_event_without_featured_image_with_description_without_cost__1.php

```diff
@@ -4,8 +4,6 @@ class="tribe-events-calendar-month__calendar-event-tooltip"
 	role="tooltip"
 >
 	<div id="tribe-events-tooltip-content-8">
-				<div class="tribe-events-calendar-month__calendar-event-tooltip-description tribe-common-b3">
```

Comment:
this test says it is rendered with description, but the snapshot does not have a description. can we fix this test to make sure we're outputting the correct snapshot?